samples/WebApp/Callback.html

```diff
@@ -0,0 +1,50 @@
+<!--
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
This line has changed since 2016; Google is a LLC. See go/releasing or look here:

```
Copyright 2018 Google LLC
```

Reply:
Done